src/Compilers/Lua/Test/Portable/Lexical/LexicalTestData.cs

```diff
@@ -100,6 +100,23 @@ public static IEnumerable<ShortToken> GetTokens(LuaSyntaxOptions options)
                     Some<object?>(value));
             }
 
+            // LuaJIT
+            foreach (var text in new[]
+            {
+                "10ULL", "20ULL", "200005ULL"
```

Comment:
It'd be nice to have a case here for a number that would overflow in signed integers such as `18446744073709551615`.